lib/core/presentation/widgets/poap/poap_claimed_poup/poap_claimed_popup.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:app/core/domain/poap/entities/poap_entities.dart';
+import 'package:app/core/domain/token/entities/token_entities.dart';
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/core/presentation/widgets/image_placeholder_widget.dart';
+import 'package:app/core/utils/media_utils.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class PoapClaimedPopup extends StatelessWidget {
+  const PoapClaimedPopup({
+    super.key,
+    this.claim,
+    this.token,
+    this.onClose,
+    this.onTransfer,
+    this.onView,
+  });
+
+  final Claim? claim;
+  final TokenDetail? token;
+  final Function()? onClose;
+  final Function()? onTransfer;
+  final Function()? onView;
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    final screenHeight = MediaQuery.of(context).size.height;
+    final t = Translations.of(context);
```

Comment:
we have `1.sh` represent for MediaQuery.of(context).size.height, you can use that instead of create a new variable

Reply:
Nice, I'm just know this also, lol :v

Reply:
👍